pyvista/plotting/plotter.py

```diff
@@ -1555,26 +1564,26 @@ def get_default_cam_pos(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
         return self.renderer.get_default_cam_pos(*args, **kwargs)
 
     @wraps(Renderer.remove_actor)
-    def remove_actor(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def remove_actor(self, *args, **kwargs) -> bool:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.remove_actor``."""
         for renderer in self.renderers:
             renderer.remove_actor(*args, **kwargs)
         return True
 
     @wraps(Renderer.set_environment_texture)
-    def set_environment_texture(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_environment_texture(self, *args, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_environment_texture``."""
         return self.renderer.set_environment_texture(*args, **kwargs)
 
     @wraps(Renderer.remove_environment_texture)
-    def remove_environment_texture(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def remove_environment_texture(self, *args, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.remove_environment_texture``."""
         return self.renderer.remove_environment_texture(*args, **kwargs)
 
     #### Properties from Renderer ####
 
     @property
-    def actors(self):  # numpydoc ignore=RT01
+    def actors(self) -> dict[str, Actor]:  # numpydoc ignore=RT01
```

Comment:
Went through one more time.  This is the only one that jumped out at me as questionable.  Should it be `vtkActor`?  Good work on trying to separate the two usages, despite my conflicting comments earlier.

Reply:
Indeed, thanks